action/config/remove.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package config
+
+import (
+	"strings"
+
+	"github.com/go-vela/cli/internal/client"
+
+	"github.com/spf13/afero"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+// Remove deletes one or more fields from the config file based off the provided configuration.
+func (c *Config) Remove() error {
```

Comment:
This is where we add the new `Remove()` function.

The purpose of this function is to remove one or more fields from the configuration file.